lib/rack/tracer.rb

```diff
@@ -11,14 +11,16 @@ class Tracer
     # @param tracer [OpenTracing::Tracer] A tracer to be used when start_span, and extract
     #        is called.
     # @param on_start_span [Proc, nil] A callback evaluated after a new span is created.
+    # @param on_span_finish [Proc, nil] A callback evaluated after a span is finished.
```

Comment:
Lets use `on_finish_span` to be consistent with `on_start_span`

Reply:
also fixed